templates/CRM/common/paymentBlock.tpl

```diff
@@ -68,17 +69,18 @@
     showHidePayment(isHide);
   }
   skipPaymentMethod();
+{/literal}{/if}{literal}
 
   CRM.$(function($) {
     function buildPaymentBlock(type) {
       var $form = $('#billing-payment-block').closest('form');
       {/literal}
-      {if $contributionPageID}
+      {if !$isBackOffice && $contributionPageID}
         {capture assign='contributionPageID'}id={$contributionPageID}&{/capture}
       {else}
-        {capture assign='pageID'}{/capture}
+        {capture assign='contributionPageID'}{/capture}
```

Comment:
I'm a bit confused by this. Is it defining an empty variable to avoid notices?

Reply:
Yes, that's what it's doing. Seems like a weird way to go about this, but I'm not aiming to change that.

Reply:
Can you explain why this variable needs to be renamed?

Reply:
So, there are five variables used to build the dataUrl at 106. Each is defined here in the same way in an if-else block that assigns the value if it exists or nothing if not.
This one was wrong, because it used PageID instead of ContributionPageID in the else. PageID is not a variable used anywhere else here. I'm just correcting that error, so that if there is no ContributionPageID, the variable is still assigned.